### PR TITLE
Add to RDoc of OrderedOptions the bang info

### PR DESCRIPTION
**Pull request by maurogeorge**

### CODE REVIEW

File: activesupport/lib/active_support/ordered_options.rb [hunk 1]
```diff
@@ -14,6 +15,13 @@ module ActiveSupport
   #   h.girl = 'Mary'
   #   h.boy  # => 'John'
   #   h.girl # => 'Mary'
+  #   h.dog  # => nil
+  #
+  # To raise a exception when the value is blank append a
```
Comment (robin850) [inline activesupport/lib/active_support/ordered_options.rb]: "an exception" also could you add a comma after "blank" ?

File: activesupport/lib/active_support/ordered_options.rb [hunk 2]
```diff
@@ -14,6 +15,13 @@ module ActiveSupport
...
+  # bang to the key name, like:
+  #
+  #   h.dog! # => KeyError is raised
```
Comment (robin850) [inline activesupport/lib/active_support/ordered_options.rb]: Could you put "# => raises KeyError" to be consistent with the existing documentation please ?

**Comment by maurogeorge** [comment]
@robin850 thanks for the review! Docs updated.

**Comment by robin850** [comment]
Thanks! :-)

**Pull request merged**